src/device-registry/bin/kafka-consumer.js

```diff
@@ -109,13 +109,13 @@ const consumeHourlyMeasurements = async (messageData) => {
             site_latitude,
             site_longitude,
           } = value[detail.path[0]];
-          return {
-            message: detail.message,
-            key: detail.context.key,
-            value: detail.context.value,
-            device_number: device_number ? device_number : undefined,
-            timestamp: timestamp ? timestamp : undefined,
-          };
+          // return {
```

Comment:
Hi @BenjaminSsempala , thanks for this.
Please shed some light on this particular change.

Reply:
This change is meant to allow the insertion to proceed for correct measurements even when there are errors in some other measurements, specifically for staging which has undeployed and test devices.

Reply:
Hi @BenjaminSsempala , why don't we then just "deploy" the devices via Staging?
I always have mixed feelings about any changes which "relax" input validations.

I suggest we leave the input validations intact since staging attempts to mirror what we wish to see in production. There was a time when we never had input validations for anything which eventually created a lot of avoidable "data cleaning work" for us.